docs/userguides/projects.md

```diff
@@ -122,8 +122,8 @@ installed, giving you more flexibility in how you define your scripts.
 
 
 Use tests to verify your project. Testing is a complex topic, learn more about testing using Ape framework
-[here](docs/userguides/testing.md)
+[here](../docs/userguides/testing.md)
 
 You can test your project using the `ape test` command. The `ape test` command comes with the core-plugin `ape-test`.
 The `ape-test` plugin extends the popular python testing framework
-[pytest](https://docs.pytest.org/en/6.2.x/contents.html).
\ No newline at end of file
+[pytest](https://docs.pytest.org/en/6.2.x/contents.html).
```

Comment:
What was the change here?

Reply:
i didn't change anything here, it just showed something changed automatically